### ISSUE TITLE
Release 0.6

### ISSUE THREAD

**Issue by m-wynn** [inejge/ldap3#27]
It would be nice to have a release with the new StartTLS features.  What else needs to be done before this release can be tagged?

**Comment by inejge** [comment]
Sorry for the delay in answering; I'm rather busy atm. So, the plan for 0.6 was to incorporate error handling overhaul, since there presently isn't a good way to gracefully handle parsing/decoding errors, but I simply haven't had time to work on it. I could, and at this point probably should, publish what's present as 0.6, and punt the rest to the next release. Version numbers aren't in short supply, after all :smiley: Expect 0.6 to appear shortly.

**Comment by inejge** [comment]
0.6 has been out for a couple of weeks. Closing this as done.
